xml/ha_migration.xml

```diff
@@ -562,24 +601,84 @@
 &prompt.root;systemctl enable sbd</screen>
     </step>
    </procedure>
-   <para/>
-   <note xml:id="note.ha.cib.upgrade">
+
+ <note xml:id="note.ha.cib.upgrade">
     <title>Upgrading the CIB Syntax Version</title>
     <para>
-     Tags (for grouping resources) and some ACL features only work with the
-     CIB syntax version <literal>pacemaker-2.0</literal> or higher. (To
-     check your version, use the <command>cibadmin -Q |grep
-     validate-with</command> command.) If you have upgraded from &productname; 11
-     SPx, your CIB version will <emphasis>not</emphasis> be upgraded by
-     default. To manually upgrade to the latest CIB version use one of the
-     following commands:
-    </para>
+     Sometimes new features are only available with the latest CIB syntax version.
+     Depending from which product version you upgraded, your CIB syntax version
+     will <emphasis>not</emphasis> be upgraded by default.
+     </para>
+    <procedure>
+     <step>
+      <para>Check your version with:</para>
+<screen>cibadmin -Q | grep validate-with</screen>
+     </step>
+     <step>
+      <para>Manually upgrade to the latest CIB syntax version with:
+      </para>
 <screen>&prompt.root;<command>cibadmin</command> --upgrade --force</screen>
-    <para>
-     or
-    </para>
-<screen>&prompt.root;<command>crm</command> configure upgrade force</screen>
```

Comment:
@tomschr Ah, I think it was removed here. I guess the purpose was to only keep a single way rather than multiple options meaning possible confusions? If so, it's fine :-)